pkg/grapicmd/internal/module/generator/template/init.go

```diff
@@ -6,57 +6,37 @@ import (
 	"github.com/jessevdk/go-assets"
 )
 
+var _Init38e76c5db8962fa825cf2bd8b23a2dc985c4513e = "*.so\n/vendor\n/bin\n/tmp\n"
 var _Init50bb4ac2099b3758964058926b3c90524e478a2c = ""
+var _Initce49182536bf5d7fb269f752e2e9d15dd5eaf7d6 = "package main\n\nimport (\n\t\"os\"\n\n\t\"google.golang.org/grpc/grpclog\"\n)\n\nfunc main() {\n\terr := run()\n\tif err != nil {\n\t\tgrpclog.Errorf(\"server was shutdown with errors: %v\", err)\n\t\tos.Exit(1)\n\t}\n}\n"
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
line is 275 characters (lll)